packages/atdgen/atdgen.1.6.0/files/atdgen.install

```diff
@@ -0,0 +1,4 @@
+bin: [
+  "atdgen.run"
+  "atdgen"
```

Comment:
The installation step is failing because these files aren't available.  Should they be copied from the `src` directory?  If so, you can fix the problem by changing them to the following:

```
  "src/atdgen.run" {"atdgen.run"}
  "src/atdgen" {"atdgen"}
```